scripts/webpack/webpack-resources.js

```diff
@@ -235,24 +235,18 @@ module.exports = {
   },
 };
 
-function getPlugins(bundleName, isProduction) {
+function getPlugins(bundleName, isProduction, profile) {
   const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 
   const plugins = [];
 
-  if (isProduction) {
+  if (isProduction && profile) {
     plugins.push(
       new BundleAnalyzerPlugin({
         analyzerMode: 'static',
         reportFilename: bundleName + '.stats.html',
         openAnalyzer: false,
-        generateStatsFile: true,
-        statsOptions: {
-          source: false,
-          reasons: false,
-          chunks: false,
-        },
-        statsFilename: bundleName + '.stats.json',
+        generateStatsFile: false,
```

Comment:
I believe the stats file is what sizeauditor consumes.

Reply:
yup. so what i realized is generating the stats file is the slow part. i moved it to size-auditor. but i don't have a solution to reduce that cost yet :(